tests/test_coor_trans.py

```diff
@@ -47,3 +51,16 @@ def testQrandRots(self):
         results = np.load(os.path.join(DATA_DIR, 'rand_rot_matrices32.npy'))
         rot_matrices32 = qrand_rots(32, seed=0)
         self.assertTrue(np.allclose(np.moveaxis(results, 2, 0), rot_matrices32, atol=1e-7))
+
+    def testRegisterRots(self):
+        angles = uniform_random_angles(32, seed=0)
+        rots_ref = Rotation.from_euler('ZYZ', angles).as_dcm()
+
+        q_ang = [[45, 45, 45]]
+        q_mat = Rotation.from_euler('ZYZ', q_ang, degrees=True).as_dcm()[0]
+        flag = 0
+        regrots_ref = get_aligned_rotations(rots_ref, q_mat, flag)
+        q_mat_est, flag_est = register_rotations(rots_ref, regrots_ref)
+
+        self.assertTrue(np.allclose(flag_est, flag)
+                        and np.allclose(q_mat_est, q_mat))
```

Comment:
above we use `atol`.  Is there any reason to believe we need `atol` here too?

Reply:
Actually default one, `atol=1e-08`, has better accuracy. Changed to use the default value for all other functions in the same file in commit  3f1e7db.